apps/src/codebridge/FileBrowser/FileBrowserHeaderPopUpButton.tsx

```diff
@@ -68,42 +67,52 @@ export const FileBrowserHeaderPopUpButton = () => {
   return (
     <>
       <FileUploaderComponent />
-      <PopUpButton
-        iconName="plus"
-        alignment="left"
-        id="uitest-files-plus"
-        ariaLabel={codebridgeI18n.manageFiles()}
-      >
-        <PopUpButtonOption
-          iconName="plus"
-          labelText={codebridgeI18n.newFolder()}
-          clickHandler={() =>
-            openNewFolderPrompt({parentId: DEFAULT_FOLDER_ID})
-          }
-        />
-        <PopUpButtonOption
-          iconName="plus"
-          labelText={codebridgeI18n.newFile()}
-          clickHandler={() => openNewFilePrompt(openNewFilePromptArgs)}
-          id="uitest-new-file"
-        />
-        <PopUpButtonOption
-          iconName="upload"
-          labelText={codebridgeI18n.uploadFile()}
-          clickHandler={() => startFileUpload()}
-        />
-        <PopUpButtonOption
-          iconName="backpack"
-          labelText={codebridgeI18n.importFromBackpackTitle()}
-          clickHandler={() =>
-            openImportFromBackpackPrompt({
-              backpackApi: backpackApi,
-              projectFiles: files,
-              validationFile: validationFile,
-            })
-          }
-        />
-      </PopUpButton>
+      <ActionDropdown
+        name="uitest-files-plus"
+        labelText={codebridgeI18n.manageFiles()}
+        triggerButtonProps={{
+          isIconOnly: true,
+          icon: {iconName: 'plus', iconStyle: 'solid'},
+          color: 'gray',
+          type: 'tertiary',
+        }}
+        size="xs"
+        menuVerticalPlacement="bottom"
+        options={[
+          {
+            value: 'newFolder',
+            label: codebridgeI18n.newFolder(),
+            icon: {iconName: 'plus'},
+            onClick: () => {
+              openNewFolderPrompt({parentId: DEFAULT_FOLDER_ID});
+            },
+          },
+          {
+            value: 'newFile',
+            label: codebridgeI18n.newFile(),
+            icon: {iconName: 'plus', iconStyle: 'solid'},
+            onClick: () => openNewFilePrompt(openNewFilePromptArgs),
+            optionId: 'uitest-new-file',
```

Comment:
why not test for label/value of the option instead of setting actual id to the option?